### PR TITLE
Add an /internal/import-table API to fetch import table data

### PR DESCRIPTION
**Pull request by juliawu**
In order to support adding an "import history" table to the /admin page of custom DCs, this PR adds a new internal API endpoint `/internal/import-table/` to fetch import history.

Testing: Added test for GetImportTableData, and accessed the new endpoint while running locally.

### CODE REVIEW

File: internal/sqldb/sqlquery/get_import_table.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	"encoding/json"
+	"time"
+
+	pb "github.com/datacommonsorg/mixer/internal/proto"
+	"github.com/datacommonsorg/mixer/internal/util"
+)
+
+// Represents "metadata" column of import table
+type ImportMetadata struct {
+	NumObs  int32
```
Comment (keyurva) [inline internal/sqldb/sqlquery/get_import_table.go:28]: Wondering if we want to allow them to be `nil` by making them pointers? We might add more metadata in the future and only a subset of fields may have values at different times.

File: proto/internal.proto [hunk 2]
```diff
@@ -40,4 +41,23 @@ message UpdateCacheRequest {
...
+
+// Request to get data in the import table
+message GetImportTableDataRequest {
+}
+
+// Response of GetImportTableDataRequest
+message GetImportTableDataResponse{
+  message ImportData {
+    message ImportMetadata{
+      int32 num_obs = 5;
```
Comment (keyurva) [inline proto/internal.proto:54]: To accommodate unset values, consider making them `optional`.

**Review by keyurva — COMMENTED** [review]
Very cool!

File: internal/sqldb/sqlquery/get_import_table.go (hunk 1 above)
Comment (juliawu) [reply internal/sqldb/sqlquery/get_import_table.go]: Good idea, made them pointers.

File: proto/internal.proto (hunk 2 above)
Comment (juliawu) [reply proto/internal.proto]: Good idea, made them `optional`.

**Review by keyurva — APPROVED** [review]

**Pull request merged**